cluster/manager.go

```diff
@@ -1080,6 +1093,9 @@ func (c *ClusterManager) Start(
 	if err != nil {
 		return err
 	}
+	if config.ConfigSchedManager == nil {
```

Comment:
can we use ```CheckNullClusterServerConfiguration()``` here instead of checking each of Manager objects?

Reply:
Yeah, I think that is a good idea. I will make the change.